src/tiled/tileselectiontool.cpp

```diff
@@ -43,6 +46,47 @@ TileSelectionTool::TileSelectionTool(QObject *parent)
     , mSelecting(false)
 {
     setTilePositionMethod(OnTiles);
+    QIcon replaceIcon(QLatin1String(":images/16x16/selection-replace.png"));
+    QIcon addIcon(QLatin1String(":images/16x16/selection-add.png"));
+    QIcon subtractIcon(QLatin1String(":images/16x16/selection-subtract.png"));
+    QIcon intersectIcon(QLatin1String(":images/16x16/selection-intersect.png"));
+
+    mReplace = new QAction(this);
+    mReplace->setIcon(replaceIcon);
+    mReplace->setCheckable(true);
+    mReplace->setChecked(true);
+    mReplace->setToolTip(tr("Replace Selection"));
```

Comment:
Please just call `languageChanged` at the end of the constructor, so you can remove all these lines here.